### PR TITLE
Add dynamic contributors' avatars to Readme Closes #5225

### LINKED ISSUE #5225: Enhance Readme with Dynamically Fetched Contributors' Avatars
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

Proposed Solution
Integrate a dynamic solution that fetches contributors' avatars from their GitHub profiles and displays them in the Readme. This will not only showcase the diverse community contributing to the project but also add a personal touch to the documentation.

### Motivation

Adding dynamically fetched contributors' avatars to the Readme file is not just a cosmetic enhancement; it's a visual celebration of our contributors' invaluable efforts. This addition fosters a stronger sense of community, encouraging both new and existing contributors to feel recognized and connected to the project. Let's make our Readme reflect the vibrant and diverse community that drives the success of our open-source endeavor.

### Example

<h2>🤝 Contributors</h2>

Thanks go to these wonderful people:

<a href="https://github.com/fastify/fastify/graphs/contributors">
  
</a>

### PR DESCRIPTION
**Pull request by aditya-armal**
Integrate a dynamic solution to fetch contributors' avatars from their GitHub profiles and display them in the Readme. This enhancement celebrates the diverse community contributing to the project, adding a personal touch to the documentation. By showcasing the faces behind the code, we aim to strengthen the sense of community and recognition for our invaluable contributors. Let's make our Readme reflect the vibrant and diverse community that drives the success of our open-source endeavor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -405,3 +405,11 @@ dependencies:
 - ISC
 - BSD-3-Clause
 - BSD-2-Clause
+
+<h2>🤝 Contributors</h2>
```
Comment (Eomm) [inline README.md:409]:
```suggestion
## Contributors
```

other titles do not have emoji

File: README.md [hunk 2]
```diff
@@ -405,3 +405,11 @@ dependencies:
...
+
+Thanks go to these wonderful people:
+
+<a href="https://github.com/fastify/fastify/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=fastify/fastify" />
```
[the remark this answers is not in the document]
Comment (aditya-armal) [reply README.md:414]: Sure.

[the remark this answers is not in the document]
Comment (aditya-armal) [reply README.md:414]: https://allcontributors.org/docs/en/overview Follow this link to get more detailed view of each contributors.

[the remark this answers is not in the document]
Comment (aditya-armal) [reply README.md:414]: I have tried to find a solution to map image with their profile, but couldn't find one. Can you please provide what can be done ahead? @jsumners

[the remark this answers is not in the document]
Comment (aditya-armal) [reply README.md:414]: https://allcontributors.org/docs/en/overview ,if we use this approach, you can map avatar's to their profile as well as their contributions as coder, bug reporter, doc, content etc. Does this approach fulfil your need?

[the remark this answers is not in the document]
Comment (Uzlopak) [reply README.md:414]: Imagine loading multiple hundred images....

**Comment by Uzlopak** [comment]
I am against this PR.

**Comment by mcollina** [comment]
I don't feel we should really land this, so I propose we close it.

Thanks for your contribution anyway!

**Pull request closed without merging**